### PR TITLE
syncthing: update to 0.14.29

### PR DESCRIPTION
**Pull request by InuSasha**

### CODE REVIEW

File: packages/addons/service/syncthing/package.mk [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 PKG_NAME="syncthing"
 PKG_VERSION="0.14.3"
```
Comment (awiouy) [inline packages/addons/service/syncthing/package.mk]: Update to current release of Syncthing.
Update changelog.txt accordingly.

File: packages/addons/service/syncthing/package.mk [hunk 2]
```diff
@@ -18,7 +18,7 @@
...
-PKG_REV="104"
+PKG_REV="105"
```
Comment (awiouy) [inline packages/addons/service/syncthing/package.mk]: Update changelog.txt accordingly.

**Comment by InuSasha** [comment]
@awiouy updated to actual latest release. 
Is the version updated needed for the LE8 pr?

**Comment by awiouy** [comment]
Thanks @InuSasha 
It would be nice to keep LE8 and LE9 packages synchronized

File: packages/addons/service/syncthing/changelog.txt [hunk 3]
```diff
@@ -1,3 +1,6 @@
+8.0.105
```
Comment (CvH) [inline packages/addons/service/syncthing/changelog.txt]: @InuSasha better change it to the new format `8.0.105` -> `105` (same for backport pr)
makes backport easier ;) (btw master is not 8.x)

**Comment by InuSasha** [comment]
bring to new changelog format.
@CvH what did you mean with "master is not 8.x"?
This one is for master, a seperate backport pr (branch libreelec-8.0 is open)

**Comment by CvH** [comment]
@InuSasha the changelog for master was pointing to 8.0.xxx and master is not 8.0 (which is fixed now if you use `105`), no need to over complicate those things :)

**Pull request merged**